### PR TITLE
added missing info (Part 2)

### PR DESCRIPTION
**Pull request by rocketsofthsv**
## Summary

@mairaw

### CODE REVIEW

**Comment by rocketsofthsv** [comment]
@mairaw just reminding you of this PR

**Comment by rocketsofthsv** [comment]
@mairaw reminding you of this PR again.

File: xml/Microsoft.Vsa/VsaLoader.xml [hunk 1]
```diff
@@ -610,8 +610,8 @@
         <Parameter Name="PDB" Type="System.Byte[]" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="PE">To be added.</param>
-        <param name="PDB">To be added.</param>
+        <param name="PE">The file formatted in PE.</param>
+        <param name="PDB">The file formatted in PDB.</param>
         <summary>This member supports the .NET Framework infrastructure and is not intended to be used directly from your code.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaLoader.xml]: We can't leave these "This member supports the .NET Framework infrastructure and is not intended to be used directly from your code." when doing these APIs. Add a summary and then add the forInternalUseOnly tag after remarks.

File: xml/Microsoft.Vsa/VsaLoader.xml [hunk 2]
```diff
@@ -610,8 +610,8 @@
...
         <remarks>To be added.</remarks>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaLoader.xml]:
```suggestion
        <remarks>To be added.</remarks>
        <forInternalUseOnly />
```

File: xml/Microsoft.Vsa/VsaLoader.xml [hunk 3]
```diff
@@ -639,7 +639,7 @@
         <Parameter Name="Site" Type="Microsoft.Vsa.IVsaPersistSite" />
       </Parameters>
       <Docs>
-        <param name="Site">To be added.</param>
+        <param name="Site">Manages project persistence and stores and retrieves code and other items using save and load operations implemented by the host.</param>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaLoader.xml]:
```suggestion
        <param name="Site">The site established by the <see cref="T:Microsoft.Vsa.VsaLoader" /> object to which source state is saved.</param>
```

File: xml/Microsoft.Vsa/VsaLoader.xml [hunk 4]
```diff
@@ -639,7 +639,7 @@
...
         <summary>This member supports the .NET Framework infrastructure and is not intended to be used directly from your code.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaLoader.xml]:
```suggestion
        <summary>Directs the script engine to persist its source state to the specified <see cref="T:Microsoft.Vsa.VsaLoader" /> object.</summary>
```

File: xml/Microsoft.Vsa/VsaLoader.xml [hunk 5]
```diff
@@ -639,7 +639,7 @@
...
         <remarks>To be added.</remarks>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaLoader.xml]:
```suggestion
        <remarks>To be added.</remarks>
        <forInternalUseOnly />
```

File: xml/Microsoft.Vsa/VsaModule.xml [hunk 6]
```diff
@@ -47,7 +47,7 @@
         <Parameter Name="bIsVsaModule" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="bIsVsaModule">To be added.</param>
+        <param name="bIsVsaModule">True if the module is a VSA module, false otherwise</param>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaModule.xml]: see how we do the formatting for other boolean params using the see langword tag

also check https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors for language style rules

File: xml/Microsoft.Vsa/VsaModule.xml [hunk 7]
```diff
@@ -47,7 +47,7 @@
...
         <summary>This member supports the .NET Framework infrastructure and is not intended to be used directly from your code.</summary>
         <remarks>To be added.</remarks>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaModule.xml]:
```suggestion
        <remarks>To be added.</remarks>
        <forInternalUseOnly />
```

File: xml/Microsoft.Vsa/VsaModule.xml [hunk 8]
```diff
@@ -47,7 +47,7 @@
         <Parameter Name="bIsVsaModule" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="bIsVsaModule">To be added.</param>
+        <param name="bIsVsaModule">True if the module is a VSA module, false otherwise</param>
         <summary>This member supports the .NET Framework infrastructure and is not intended to be used directly from your code.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Vsa/VsaModule.xml]: Check how we write summaries
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Constructor

File: xml/Microsoft.Vsa.Vb.CodeDOM/CodeDOMProcessor.xml [hunk 9]
```diff
@@ -75,9 +75,9 @@
         <Parameter Name="xmlStream" Type="System.String" />
       </Parameters>
       <Docs>
```
[the remark this answers is not in the document]
Comment (rocketsofthsv) [reply xml/Microsoft.Vsa.Vb.CodeDOM/CodeDOMProcessor.xml]: Guess

**Pull request merged**